### PR TITLE
Updates to hardening intro topic

### PR DESCRIPTION
**Pull request by Charlotte-Holt**

### CODE REVIEW

File: ref/general/hardening-intro.adoc [hunk 1]
```diff
@@ -37,26 +42,29 @@ To learn more about OS vulnerabilities and ways to prevent OS intrusions, go to
...
 * *Enable the Transport Security feature*: The link:/docs/ref/feature/#transportSecurity.html[Transport Security feature] enables the link:/docs/ref/feature/#ssl.html[SSL feature] to support SSL connections and adds the ability to configure outbound transports. Additionally, disable non-SSL ports by setting them to `-1` in the `server.xml` file, and use only certificates that are signed by a certificate authority for public-facing production servers. Also, enable SSL in your client and web server. Using SSL connections throughout your network communication flow helps protect your server from network intrusions.
 +
-If you use SSO for user authentication, all SSO endpoint URLs must use an HTTPS scheme, and the client must perform a TLS server certificate check.
-* *Run applications on a virtual host*: Configure applications to run on a virtual host. Open Liberty admin traffic comes in on the default host, so configuring your applications to run on a virtual host keeps Open Liberty admin traffic separate from application requests.
+If you use SSO for user authentication, all SSO endpoint URLs must use an HTTPS scheme, and the client must perform an SSL server certificate check.
```
Comment (ManasiGandhi) [inline ref/general/hardening-intro.adoc]: In this sentence "If you use SSO for user authentication, all SSO endpoint URLs must use an HTTPS scheme, and the client must perform an SSL server certificate check." avoid the use of the word 'all'. You can say "the SSO endpoint URLs must use an HTTPS scheme,". Check with the SME for technical accuracy.

File: ref/general/hardening-intro.adoc [hunk 2]
```diff
@@ -37,26 +42,29 @@ To learn more about OS vulnerabilities and ways to prevent OS intrusions, go to
...
+// Does this apply to cloud-native microservice scenarios?
+// * *Run applications on a virtual host*: Configure applications to run on a virtual host. Open Liberty admin traffic comes in on the default host, so configuring your applications to run on a virtual host keeps Open Liberty admin traffic separate from application requests.
```
Comment (ManasiGandhi) [inline ref/general/hardening-intro.adoc]: "Open Liberty admin traffic comes in on the default host, " here check if it's okay to use  the term 'admin'.

File: ref/general/hardening-intro.adoc (hunk 1 above)
Comment (Charlotte-Holt) [reply ref/general/hardening-intro.adoc]: I think I'll keep all and check with the SME on this. I think his point was to stress that all SSO endpoint URLs must use it, but I can see if he's okay with me taking it out.

File: ref/general/hardening-intro.adoc (hunk 2 above)
Comment (Charlotte-Holt) [reply ref/general/hardening-intro.adoc]: Can check with SME to make sure terminology is correct.

File: ref/general/hardening-intro.adoc (hunk 1 above)
Comment (ManasiGandhi) [reply ref/general/hardening-intro.adoc]: Okay.

**Pull request merged**